RELEASENOTE.md

```diff
@@ -1,8 +1,34 @@
 Version Notes (Current Version: v1.3)
 =======================================
-v1.3 (Pre-release)
+v1.3
 -----------
-
+1. `docfx` improvements
+    1. Add subcommand `docfx template`. You can now `docfx template list` and `docfx template export -A` to list and export all the embeded templates!
+    2. Add subcommand `docfx merge`. You can use this subcommand to merge `platform` from multiple APIs with the same `uid`
+    3. Add two options to `build` subcommand, `--exportRawModel` and `--exportViewModel`. `--exportRawModel` exports the data model to apply templates, `--exportViewModel` exports the view model after running template's pre-process scripts.
+    4. Add `--globalMetadata`, and `--globalMetadataFile` options to `build` subcommand. These options allow `globalMetadata` to be loaded from command line in json format or from a JSON file.
+    5. Add `--fileMetadataFile` option to `build` subcommand. This option allows `fileMeatdata` to be read from an external JSON file.
+    6. Support plugins. You can create your own template with a `plugins` folder, inside which, you create your own build steps. Refer to @Microsoft.DocAsCode.EntityModel.Plugins.BaseDocumentBuildStep for a sample plugin implementation. 
+2. *DFM* syntax improvements
+    1. Support note&div syntax
+    2. Support *query* format in *code snippet*
+       `[!code-<language>[<name>](<codepath><queryoption><queryoptionvalue> "<title>")]`
+    3. Change *xref* logic:
+        1. If content after `@` is wrapped by `'` or `"`,  it contains any character including white space
+        2. If content after `@` is not wrapped by `'` or `"`, it ends when
+            a. line ends
+            b. meets whitespaces
+            c. line ends with `.`, `,`, `;`, `:`, `!`, `?` and `~`
+            d. meets 2 times or more `.`, `,`, `;`, `:`, `!`, `?` and `~`
```

Comment:
Not display correctly in preview. Seems that using "1234" instead of "abcd" can work.